src/less/properties/ds6/progress-spinner-properties.less

```diff
@@ -0,0 +1,21 @@
+@import "../../variables/ds6/progress-spinner-variables.less";
+
+.progress-spinner {
+    --progress-spinner-image-url: @progress-spinner-image-url;
+}
+
+.progress-spinner--large {
+    --progress-spinner-large-image-url: @progress-spinner-large-image-url;
```

Comment:
@ianmcburnie question... usually we put all our custom properties in the base class. Large won't be used except in this case, but should it still live in the base class since that base class is needed for spinner to work?

In all our properties we have the variables all assigned to the base class even if they are for different cases, like we have `--button-primary` variable in `button
So I think we should be consistent in that regards

Reply:
Yes, I overlooked that. Thanks, Andrew.

Reply:
done